benten-hackernews-bolt/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target/
```

Comment:
Other bolts included their target dir, do you want mew to include the built bolt as well?

Reply:
@jakeleveroni this is fine, the right thing to do. I will exclude this for other bolts as well.